lms/djangoapps/courseware/access_utils.py

```diff
@@ -80,3 +80,12 @@ def in_preview_mode():
     hostname = get_current_request_hostname()
     preview_lms_base = settings.FEATURES.get('PREVIEW_LMS_BASE', None)
     return bool(preview_lms_base and hostname and hostname.split(':')[0] == preview_lms_base.split(':')[0])
+
+
+def is_preview_menu_disabled(user, course):
```

Comment:
Nit: I know you'd have to add nots, but I prefer positive like `is_preview_menu_enabled`.